src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -397,32 +1032,33 @@ function NetCDFOutputWriter(model, outputs;
             @warn "Overwriting existing $filepath."
         end
     end
-    # TODO: This call to dictify is only necessary because "dictify" is hacked to help
-    # with LagrangianParticles output (see the end of the file).
-    # We shouldn't support this in the future; we should require users to 'name' LagrangianParticles output.
-    outputs = dictify(outputs)
-    outputs = Dict(string(name) => construct_output(outputs[name], grid, indices, with_halos) for name in keys(outputs))
+
+    outputs = Dict(
+        string(name) => construct_output(outputs[name], grid, indices, with_halos)
+        for name in keys(outputs)
+    )
```

Comment:
Probably we should follow the conventions we already use in Oceananigans?

```suggestion
outputs = Dict(string(name) => construct_output(outputs[name], grid, indices, with_halos)
               for name in keys(outputs))
```

Or in this case it might be better to even do a one-liner.